### DISCUSSION TITLE
Removing part of sprites

### DISCUSSION THREAD

**Discussion by Nikv2005** [Monika-After-Story/MonikaModDev#9247] [QA - code and content creation (technical)] [answered]
There is a way to remove for example hand of Monika in some moments? I mean when some "dialog" starts is possible to remove her hand or another part of body? Make it just disappeared.

**Comment by ThePotatoGuy** [comment]
Why should sprites be removed for certain dialogue? Can you give an example of the problem?

**Comment by Nikv2005** [comment]
> Why should sprites be removed for certain dialogue? Can you give an example of the problem?

Well, imagine that there a dialog , but at some moment you need that Monika's hand gone , i mean in game , i just need  her hand gone , and after dialog hand back again.

**Comment by Booplicate** [comment]
Can you post part of your dialogue?
By "gone" you mean move it to the right/down?

**Comment by Nikv2005** [comment]
> Can you post part of your dialogue?
> By "gone" you mean move it to the right/down?

Hmmm... So imagine there is a dialog and in needed time her hand disappeared , just disappear . Maybe it moves to somewhere , but i shouldn't see her hand . Imagine Monika without right hand , that what should happen.

**Comment by Booplicate** [comment, marked as answer]
I assume you mean an expression and not just completely hiding part of her sprites, check out [expression previewer](https://github.com/Monika-After-Story/MonikaModDev/blob/master/FAQ.md#how-do-i-find-the-spritecode-for-an-expression). Although, don't use your main install while testing your code.
